src/app/components/mno-apps/mno-app.html

```diff
@@ -117,7 +117,7 @@ <h2>{{ 'mno_enterprise.templates.dashboard.marketplace.show.price' | translate }
               <b>{{ 'mno_enterprise.templates.dashboard.marketplace.show.free_trial_duration' | translate }}</b>: {{vm.app.free_trial_duration}} {{vm.app.free_trial_unit}}
             </li>
             <li ng-repeat="plan in vm.pricing_plans">
-              <b>{{plan.name}}</b>: {{plan.description}}
+              <b>{{plan.name}}</b>: {{plan.description}}<b>   <i ng-show="plan.quote_based" class="fa fa-question-circle" uib-tooltip="{{ 'mno_enterprise.templates.dashboard.marketplace.show.quote_based' | translate }}"></i>
```

Comment:
I think you have an extra `<b>` here right after `{{plan.description}}` and some extra spaces after that (looks like 3 spaces before the opening `<i>` tag).